### DISCUSSION TITLE
Response output parsing

### DISCUSSION THREAD

**Discussion by zhirnov-av** [RobinTail/express-zod-api#354] [Q&A] [answered]
Hello!
Is it possible to exclude output object parsing?
We have performance issues with it.
For example:
- collecting data from db: ~46ms
- transform to response object: ~30ms
- request handler total time: ~100ms
- output parsing in express-zod-api: ~1285ms

I talking about this method in endpoint.ts:
```
  async #parseOutput(output: any) {
    try {
      return await this.outputSchema.parseAsync(output);
    } catch (e) {
      if (e instanceof z.ZodError) {
        throw new z.ZodError([
          {
            message: "Invalid format",
            code: "custom",
            path: ["output"],
          },
          ...e.issues.map((issue) => ({
            ...issue,
            path: issue.path.length === 0 ? ["output"] : issue.path,
          })),
        ]);
      }
      throw e;
    }
  }
```

Thank you.

**Comment by RobinTail** [comment, marked as answer]
Hello @zhirnov-av.

Thank you for the question.
The possible obstacle for such feature is that output schema may contain transformations, therefore passing the endpoint's handler output through the schema might be required.

Could you please provide a bit more details:
- what is the version of express-zod-api installed?
- what is the version of NodeJS?
- what is the output schema you made the benchmark on?

Looking forward to your reply.
